bin/start.sh

```diff
@@ -40,6 +40,53 @@ cd ${ZLUX_APP_SERVER_DIR}/bin
 . ./init/node-init.sh
 . ./utils/setup-logs.sh
 
+
+###################################
+# Start cleanup of node IPC MQ
+###################################
+# node.js instance is not fully cleaned up when exits. As time going, the message
+# queue will be full and any node.js command will generate this error:
+#
+# msgget: EDC5133I No space left on device. (errno2=0x07050305)
+# CEE5207E The signal SIGABRT was received.
+# Ended with rc=131
+#
+# FIXME: this is a temporary workaround suggested by node.js team.
+# export __IPC_CLEANUP=1
+#
+# Always export __IPC_CLEANUP=1 caused another problem which the node.js process
+# may randomly hang on __getipc().
+#
+# This is proper way to cleanup IPC message queues.
+if [ "${ZWE_RUN_ON_ZOS}" = "true" ]; then  
+  id=$(id -nu)
+  # Trying to capture columns T, ID, and second to last column, which for q=LSPID, m=CPID
+  for s in $(ipcs -a | awk 'match($1,"q|m") && $5 == "'${id}'" {print $1","$2":"$(NF-1)}'); do
+    x=${s%%:*}
+    pid=${s##*:}
+    type=${x%%,*}
+    num=${x##*,}
+    if [[ $pid -gt 0 ]]; then
+      kill -0 "$pid" 1>/dev/null 2>&1
+      if [ $? -eq 0 ]; then
+        true
+      else
+        ipcrm -$type $num
+      fi
```

Comment:
Why this strange true statement, is there any special reason for this? 
```shell
if [ $? -ne 0 ]; then
  ipcrm -$type $num
fi
```

Reply:
I copied it from the old script, agreed we can change the condition to `-ne`